lib/matplotlib/pyplot.py

```diff
@@ -2991,10 +2991,10 @@ def hexbin(x, y, C=None, gridsize=100, bins=None, xscale='linear',
 # This function was autogenerated by boilerplate.py.  Do not edit as
 # changes will be lost
 @_autogen_docstring(Axes.hist)
-def hist(x, bins=None, range=None, normed=False, weights=None, cumulative=False,
-         bottom=None, histtype='bar', align='mid', orientation='vertical',
-         rwidth=None, log=False, color=None, label=None, stacked=False,
-         hold=None, data=None, **kwargs):
+def hist(x, bins=None, range=None, density=None, normed=None, weights=None,
```

Comment:
`lib/matplotlib/pyplot.py` is an autogenerated file, so these changes will need to be reverted.